File: mobile/src/main/java/treehou/se/habit/tasker/reciever/IFireReciever.java
package treehou.se.habit.tasker.reciever;

import android.content.Context;
import android.os.Bundle;

public interface IFireReciever {

    String BUNDLE_EXTRA_TYPE    = "treehou.se.habit.extra.TYPE";

    boolean fire(Context context, Bundle bundle);
}

File: mobile/src/main/java/treehou/se/habit/tasker/reciever/IFireReciever.kt
package treehou.se.habit.tasker.reciever

import android.content.Context
import android.os.Bundle

interface IFireReciever {

    fun fire(context: Context, bundle: Bundle): Boolean

    companion object {

        val BUNDLE_EXTRA_TYPE = "treehou.se.habit.extra.TYPE"
    }
}
